gobblin-utility/src/main/java/org/apache/gobblin/util/WriterUtils.java

```diff
@@ -170,6 +175,19 @@ private static WriterFilePathType getWriterFilePathType(State state) {
   }
 
   /**
+   * Creates {@link Path} for case {@link WriterFilePathType#DB_TABLE} with configurations
+   * {@link ConfigurationKeys#EXTRACT_NAMESPACE_NAME_KEY} and {@link ConfigurationKeys#EXTRACT_TABLE_NAME_KEY}
+   * @param state
+   * @return a path
+   */
+  public static Path getDbTableWriterFilePath(State state) {
+    Preconditions.checkArgument(state.contains(ConfigurationKeys.EXTRACT_NAMESPACE_NAME_KEY));
+    Preconditions.checkArgument(state.contains(ConfigurationKeys.EXTRACT_TABLE_NAME_KEY));
+    return new Path(state.getProp(ConfigurationKeys.EXTRACT_NAMESPACE_NAME_KEY) + Path.SEPARATOR + state
```

Comment:
What does EXTRACT_NAMESPACE_NAME_KEY mean in reality? Is it a DB name? or the DB name would be included in the topic.name? It's better to use specific name like EXTRACT_DB_NAME_KEY if possible because EXTRACT_NAMESPACE_NAME_KEY is something like "com.linkedin" or "gobblin.copy".

Reply:
EXTRACT_NAMESPACE_NAME_KEY defines a general domain of a table. It can be used as a DB name. A topic name is not necessary a fully qualified table name. A fully qualified namespace can have parents, like "com.linkedin.dbName". So we don't need to create a specific configuration key for DB, using the general one is good enough. 

To avoid confusing, I'll rename the file path type to be `NAMESPACE_TABLE`.